helper/response.js

```diff
@@ -0,0 +1,41 @@
+const _ = require('lodash');
+
+let success = (response, data) => {
+
+    let successResponse = {
+        success: true
+    };
+
+    if (data) {
+        _.extend(successResponse, {
+            data: data
+        });
+    }
+
+    response.status(200).json(successResponse);
+};
+
+let internalServerError = (response, err) => {
+    let message = [
+        'Error in handling this request. ',
+        'Please contact system admin.'
+    ].join('');
```

Comment:
Like the structured way of concatinating strings, but really prefer the new ES6 template literals:

https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals

The error message isn't that long, and can be written in one line :)